src/index.html

```diff
@@ -15,8 +15,79 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <ul>
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="img"
+          />
```

Comment:
The `img` tag should not be a child of the `ul` tag. The only children of the `ul` tag should be `li` tags. Please, move the `img` tag outside of the `ul` tag or wrap it in an `li` tag.